Content.Shared.Database/LogType.cs

```diff
@@ -95,4 +95,5 @@ public enum LogType
     /// </summary>
     ChatRateLimited = 87,
     AtmosTemperatureChanged = 88,
+    RoboticsConsole = 89
```

Comment:
adding logtypes has extra considerations so i'd want @Chief-Engineer 's opinion on this.

Reply:
yeah nfi about logtypes

Reply:
using Action for both so its easier to look up, and explosion logs will use Damage probably for the actual damage itself